packages/roosterjs-editor-dom/lib/list/VListItem.ts

```diff
@@ -207,9 +207,14 @@ export default class VListItem {
      */
     outdent(preventItemRemoval: boolean = false) {
         const expectedLength = preventItemRemoval ? 2 : 1;
-        if (this.listTypes.length > expectedLength) {
+        const listTypesLength = this.listTypes.length;
+        if (listTypesLength > expectedLength) {
             this.listTypes.pop();
         }
+
+        if (this.listTypes[listTypesLength - 1] == ListType.Ordered) {
+            this.node.style.listStyleType = '';
```

Comment:
why set to empty? What if the parent level list also have a list style type, should we also follow?

Reply:
Removed this part of code as the scope of this PR is only going to be pasting from Word Online.
However, I found that if a list have a 'custom' list style type and we indent or outdent, Vlist is going to remove the list style and will have a different list style. Adding a gif to explain better:

![WordPast2](https://user-images.githubusercontent.com/8291124/186755737-8df7eec8-e40e-4ec9-b608-c945814a983e.gif)

Reply:
So this should be a separate issue, right?
Does it repro without this change?

Reply:
Yes, separate issue.

Reply:
ok. please open issue to track.